coordinatorlib/src/main/scala/com/socrata/datacoordinator/truth/universe/sql/SqlTableCleanup.scala

```diff
@@ -28,4 +30,11 @@ class SqlTableCleanup(conn: Connection, daysDelay: Int = 1) extends TableCleanup
       }
     }
   }
+
+  private def cleanupDeleteds(): Unit = {
```

Comment:
Might be worth naming this `cleanupDeletedCollocations` or something a bit more specific.

Reply:
I am thinking ahead about possible future types.